### PR TITLE
Fix sign out step definition

### PR DESCRIPTION
**Pull request by jsugarman**
`messaging.feature` is failing on circleCI
almost consistently, but not locally :(

### CODE REVIEW

File: db/seeds/scheme_11.rb [hunk 1]
```diff
@@ -54,7 +54,7 @@ class CSV::Row
 
 # regenerate unique codes based on offence description and band where order is significant
 require Rails.root.join('lib','data_migrator','offence_unique_code_migrator')
-offences = agfs_fee_scheme_eleven.offences.joins(:offence_band).where(offence_class: nil).unscope(:order).order('offences.description COLLATE "C", offences.contrary COLLATE "C", offence_bands.description COLLATE "C"')
+offences = agfs_fee_scheme_eleven.offences.joins(:offence_band).where(offence_class: nil).unscope(:order).order(Arel.sql('offences.description COLLATE "C", offences.contrary COLLATE "C", offence_bands.description COLLATE "C"'))
```
Comment (jsugarman) [inline db/seeds/scheme_11.rb:57]: unrelated change to remove a warning following rails 5.2.2 upgrade

File: features/support/user_helper.rb [hunk 2]
```diff
@@ -1,6 +1,14 @@
 module UserHelper
   def sign_in(user, password)
-    visit new_user_session_path
+    # using_wait_time 10 do
```
Comment (jsugarman) [inline features/support/user_helper.rb]: remove

**Review by cleargif — APPROVED** [review]
👍
